backend/LexBoxApi/Controllers/LoginController.cs

```diff
@@ -233,6 +234,7 @@ public async Task<ActionResult> ResetPassword(ResetPasswordRequest request)
         var user = await lexBoxDbContext.Users.FindAsync(lexAuthUser.Id);
         if (user == null) return NotFound();
         user.PasswordHash = PasswordHashing.HashPassword(passwordHash, user.Salt, true);
+        await userService.SetPasswordStrength(user.Id, request.PasswordStrength);
```

Comment:
same here

Reply:
Same reply; I'll write a Clamp method and call it, saving a DB round-trip.

Reply:
Done in commit 9ca48737.